Tools/WinMLRunner/BindingUtilities.h

```diff
@@ -315,6 +343,112 @@ namespace BindingUtilities
         return ImageFeatureValue::CreateFromVideoFrame(videoFrame);
     }
 
+    ITensor CreateBindableTensorFromImage(
+        const ILearningModelFeatureDescriptor& description,
+        InputDataType inputDataType,
+        const CommandLineArgs args
+    )
+    {
+        auto name = description.Name();
+        const TensorFeatureDescriptor& tensorDescription = description.try_as<TensorFeatureDescriptor>();
```

Comment:
probably shouldnt be a reference here.
should be tensorDescriptor instead of tensorDescription.

auto tensorDescriptor = description.try_as<TensorFeatureDescriptor>();

Reply:
Amusingly I changed this to tensorDescription from tensorDescriptor to better match the description argument, which was a copy from another function in the code.  I'll happily change it back.